### PR TITLE
Ability to archive or delete selected runs

### PR DESCRIPTION
**Pull request by VkoHov**
- Add multi-select functionality in `Table`
- Implement archive and delete batch functionality in explores tables

### CODE REVIEW

**Comment by VkoHov** [comment]
Resolves [#1185](https://github.com/aimhubio/aim/issues/1185)

File: aim/web/ui/src/components/Table/DeleteModal.tsx [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    let finishedList: any[] = [];
+    let inProgressList: any[] = [];
+    const runHashList: string[] = [];
+    Object.values(selectedRows || {}).forEach((selectedRow: any) => {
+      if (!runHashList.includes(selectedRow.runHash)) {
+        runHashList.push(selectedRow.runHash);
+        const rowData = {
+          key: selectedRow.runHash,
+          run: `${moment(selectedRow.creation_time * 1000).format(
+            'DD MMM YYYY, HH:mm A',
```
Comment (Hamik25) [inline aim/web/ui/src/components/Table/DeleteModal.tsx:94]: Use constant defined date-time format.

File: aim/web/ui/src/components/Table/DeleteModal.tsx [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+              ref={tableRef}
+              fixed={false}
+              columns={tableColumns}
+              minHeight={'100px'}
+              data={data}
+              hideHeaderActions
+              headerHeight={28}
+              emptyText='No Text'
+              rowHeight={24}
+              onRowHover={() => {}}
```
Comment (Hamik25) [inline aim/web/ui/src/components/Table/DeleteModal.tsx:160]: Can be removed.

File: aim/web/ui/src/components/Table/DeleteModal.tsx [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+                ref={disabledTableRef}
+                fixed={false}
+                columns={tableColumns}
+                minHeight={'100px'}
+                data={disabledData}
+                hideHeaderActions
+                headerHeight={28}
+                emptyText='No Text'
+                rowHeight={24}
+                onRowHover={() => {}}
```
Comment (Hamik25) [inline aim/web/ui/src/components/Table/DeleteModal.tsx:183]: Can be removed.

File: aim/web/ui/src/components/kit/Modal/Modal.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+          <Button className='Modal__footer__okCancel' onClick={onClose}>
+            {cancelButtonText ?? ''}
+          </Button>
+          <Button
+            onClick={onOk}
+            variant='contained'
+            className={'Modal__footer__okButton'}
+            style={{ background: okButtonColor }}
+          >
+            {okButtonText ?? ''}
```
Comment (Hamik25) [inline aim/web/ui/src/components/kit/Modal/Modal.tsx:73]: Define default Ok text for okButtonText prop.

File: aim/web/ui/src/components/kit/Modal/Modal.tsx [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+              ) : (
+                title
+              )}
+            </div>
+          )}
+        </div>
+        <div className='Modal__content'>{children}</div>
+        <div className='Modal__footer'>
+          <Button className='Modal__footer__okCancel' onClick={onClose}>
+            {cancelButtonText ?? ''}
```
Comment (Hamik25) [inline aim/web/ui/src/components/kit/Modal/Modal.tsx:65]: Define default Cancel text for cancelButtonText prop.

**Review by arsengit — APPROVED** [review]

**Review by roubkar — APPROVED** [review]

**Pull request merged**